Purchases/Public/RCPurchasesErrorUtils.m

```diff
@@ -150,7 +150,7 @@ static RCPurchasesErrorCode RCPurchasesErrorCodeFromRCBackendErrorCode(RCBackend
     return RCUnknownError;
 }
 
-#if TARGET_OS_IPHONE
+#if TARGET_OS_IOS
```

Comment:
what about tv os?

Reply:
updated. This class actually merits more thought (maybe on a separate PR) because some of these error codes are now mac compatible